gnome-initial-setup/gnome-initial-setup.c

```diff
@@ -156,6 +158,81 @@ destroy_pages_after (GisAssistant *assistant,
   }
 }
 
+static gint
+compare_pages_order (gconstpointer a,
+                     gconstpointer b,
+                     gpointer data)
+{
+  GHashTable *sort_table = data;
+  const gchar *id_a = ((PageData *) a)->page_id;
+  const gchar *id_b = ((PageData *) b)->page_id;
+  gint order_a = (gint) GPOINTER_TO_UINT (g_hash_table_lookup (sort_table, id_a));
```

Comment:
Why use `GPOINTER_TO_UINT` then cast to `gint`? Just use `GPOINTER_TO_INT` (and `GINT_TO_POINTER` when building the hash table).